packages/vue-components/src/utils/utils.js

```diff
@@ -159,3 +161,40 @@ export function VueFixer (vue) {
   vue.mixins.unshift(mixin)
   return vue
 }
+
+export function resizeHeader(isNavMenuShowing) {
```

Comment:
hmm not sure if we should be putting this within `navbar.vue`. (note a header isn't limited to just a navbar)

how about adding the window listener in `index.js` where the fixed header padding is originally setup?
It should keep these concerns closer.

Also, could override the selectors' properties (by cascading) instead querying every element

Reply:
> Also, could override the selectors' properties (by cascading) instead querying every element

Sorry I don't really get what you meant by this 😅. Do you mean inserting new CSS rules to target these classes?

Reply:
yup

Reply:
Thanks for the clarification :)

> hmm not sure if we should be putting this within navbar.vue. (note a header isn't limited to just a navbar)
>
> how about adding the window listener in index.js where the fixed header padding is originally setup?
> It should keep these concerns closer.

 One more thing, the reason why I placed this in `navbar` is because a header padding resize is only necessary if the `lower-navbar` slot is present. In this case, is it ok to let the listener stay in `navbar`?

Reply:
> One more thing, the reason why I placed this in navbar is because a header padding resize is only necessary if the lower-navbar slot is present. In this case, is it ok to let the listener stay in navbar?

not necessarily, e.g.

`<header><box dismissible>...</box></header>` -> close box -> `fixed-header-padding` is not changed (everything appears pushed downward)

It's part of a larger generic issue (not listed anywhere) that can be dealt with separately, or in this PR since its a prerequisite (preferably a separate commit)

Reply:
Ahh thanks, wasn't aware of this issue. Added a listener in `index.js` to resize the header if it changes. Should be fixed now :)